### ISSUE TITLE
Creating a new slider

### ISSUE THREAD

**Issue by c0sm1n87** [ThemeFuse/Unyson#673]
Hello,

I followed your instructions from the dev-manual to create my own slider but for some reason, I couldn't see it in the backend.

More information on the scenario, I am not starting from the scratch template ( I merely copied what I needed from there ) but I copied the framework_customization folder and started tinkering it.

I saw that in

mytheme\framework-customizations\extensions\media\extensions\slider\extensions

I had the overrides for the available sliders so, maybe here is where it went wrong, i created another folder with my own data (keeping the syntax from the dev-manual regarding the folder name and of the php file) and so on.

Regardless of this, it didn't appear in the backend (Sliders extensions). I thought maybe I should put it in the framework's folder:
\wp-content\plugins\unyson\framework\extensions\media\extensions\slider\extensions

but with zero results as well.

The bottom line is that I overrided owl-carousel for my own needs (in the theme's folder), it's working, but I would rather do it the proper way.

PS: Imi place foarte mult frameworkul - sunt incepator in ce priveste WordPress`ul dar documentatia voastra a fost mai mult decat de ajuns sa "get it started" :) Gj

**Comment by llemurya** [comment]
@c0sm1n87 It is a small trick, you must disable and enable the slider from the Unyson and it must works and show your new created slider, this is the [video](https://youtu.be/Fx23yQrnzMo) as a tutorial how to create a slider, also see this [printscreen](http://static.md/18f92ead16d4801b80dba86c84bdb41d.png).

**Comment by c0sm1n87** [comment]
Thank you @llemurya  ! :+1:  

I appreciate going the extra mile by creating a youtube video :D

**Comment by giathinh910** [comment]
@llemurya  :+1:
